configure.ac

```diff
@@ -1077,7 +1077,7 @@ AC_CACHE_SAVE
 # visible again
 #
 ###########################################################
-OPAL_SETUP_FT_OPTIONS
+#OPAL_SETUP_FT_OPTIONS
```

Comment:
We decided the second pass will remove everything (and I suppose the macro was kept to mark it as a starting point for the second effort).